Makefile.am

```diff
@@ -3,6 +3,10 @@ AM_CFLAGS += $(CODE_COVERAGE_CFLAGS)
 AM_CFLAGS += $(SQLITE_CFLAGS) $(UV_CFLAGS) $(PTHREAD_CFLAGS)
 AM_LDFLAGS = $(UV_LIBS) $(PTHREAD_LIBS)
 
+if DQLITE_NEXT_ENABLED
+AM_CFLAGS += -DDQLITE_NEXT
```

Comment:
[Nit]: Do we normally indent inside conditionals, I can see examples of either way below.

Reply:
It seems that `if` needs to be at the start of a line, so we cannot have proper indendation for conditionals within conditionals. Given that I'd prefer to have no indentation at all; I'll harmonize everything on that style in a new commit. (Did I mention I don't like Autotools?)